fuzz/src/full_stack.rs

```diff
@@ -299,6 +299,14 @@ impl<'a> MoneyLossDetector<'a> {
 	}
 
 	fn connect_block(&mut self, all_txn: &[Transaction]) {
+		if self.blocks_connected > 50_000 {
```

Comment:
Connecting blocks looks like it's interspersed with other fuzz input commands. So now we'll just disallow connecting *any* blocks after 50k total have been connected? It seems like what we want is more like "allow connecting up to X blocks in a row, then skip connecting any more until we get a fuzz input that does something else"?

Reply:
I guess it just seems like a lot of operations will be limited if no blocks can be connected, particularly channel opens/closes. How can you tell fuzz coverage wasn't reduced?